contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -98,3 +98,28 @@ def bin_executables(self):
       return {self.package_name: self.payload.bin_executables}
 
     return self.payload.bin_executables
+
+  @property
+  def package_manager(self):
+    """
+
+    :rtype: string
```

Comment:
Would be great to use official type hints for all this code. I'll give a couple as examples, and happy to talk over Slack if you want help figuring out others.

We don't have a ton of examples in codebase because we could only do this as of a few weeks ago.

```python
@property
def package_manager(self) -> str:
  ...
```

--

For any of the files in this PR, if you are able to successfuly run `./pants mypy path/to:target` on them with no errors, then please add to that target's `BUILD` entry `tags = {'type_checked'}` for us to start enforcing type safety on that file in CI :)

Reply:
I agree completely.